pkg/job/blueocean.go

```diff
@@ -89,6 +97,39 @@ func (c *BlueOceanClient) GetBuild(option GetBuildOption) (*PipelineBuild, error
 	return &pb, nil
 }
 
+// PipelineRun represents a PipelineRun of Jenkins BlueOcean
+type PipelineRun struct {
```

Comment:
Please take a look at PipelineBuild:

https://github.com/jenkins-zh/jenkins-client/blob/8d848a9886c2aeceec09de5c147d4949c1f0e9c5/pkg/job/blueocean.go#L140-L167

I prefer to use it or let `PipelineRun` embed in `PipelineBuild`.

Reply:
My bad. I think they are the same thing. We can use one of them. But I cannot see `PipelineBuild` was used in any BlueOcean documents.

Reply:
You are right. In blueocean source code, it is called [PipelineRun](https://github.com/jenkinsci/blueocean-plugin/blob/master/blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineRunImpl.java).

Reply:
So, I'm going to rename `PipelineBuild` to `PipelineRun` if you think it's ok.

Reply:
Make sense to me

Reply:
By the way, I'm wondering if it's possible that we can put this projects' link into [BlueOcean restful document](https://github.com/jenkinsci/blueocean-plugin/tree/master/blueocean-rest) once we covered enough APIs.

Reply:
That would be nice. And I will follow this approach in further PR.